src/components/Navbar.tsx

```diff
@@ -139,6 +139,9 @@ export default function Navbar() {
             py-4 gap-8
             lg:px-0 bg-blue lg:bg-transparent lg:ml-auto
           ">
+            <Group links={[
```

Comment:
Shouldn't this just be a normal link instead of a group of links?

Reply:
@mathiasmagnusson 
Ah, but it looks like "about us" will also be there, but it can be changed into group of links then.